trade-finance-manager-ui/templates/_partials/case_sub_navigation.njk

```diff
@@ -1,6 +1,6 @@
 {% from "moj/components/primary-navigation/macro.njk" import mojPrimaryNavigation %}
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full govuk-!-padding-top-8">
+    <div class="govuk-grid-column-full  {{ 'govuk-!-padding-top-8' if not successMessage }} ">
```

Comment:
This is a stylistic change to remove the padding above the tabs on the case page when the success message is visible to keep in line with the designs